docs/docs/alephium/entrypoints.md

```diff
@@ -0,0 +1,635 @@
+---
+title: Entrypoints
+
+slug: /alephium/entrypoints
+---
+
+This section outlines the core entrypoints for the Invariant smart contract, providing developers with essential methods to interact with the protocol. These entrypoints cover various aspects of the contract, including protocol fee management, fee tier administration, pool creation and management, position handling, and swap functionality.
+
+## Deployment
+
+```rust
+async function deployInvariant(
+  signer: SignerProvider,
+  protocolFee: bigint
+): Promise<InvariantInstance>;
+```
+
+This deployment method initializes the contract with the specified protocol fee. The administrator role is assigned to the signer.
+
+#### Input parameters
+
+| Name         | Type       | Description                          |
+| ------------ | ---------- | ------------------------------------ |
+| signer | SignerProvider | The user who will get administrative access. |
+| protocolFee | bigint | The percentage for the protocol fee. |
+
+## Protocol fee
+
+### Get protocol fee
+
+```rust
+pub fn Invariant::getProtocolFee() -> U256;
+```
+
+This method retrieves the current protocol fee percentage.
+
+#### Output parameters
+
+| Type       | Description               |
+| ---------- | ------------------------- |
+| U256 | The current protocol fee. |
+
+### Withdraw protocol fee
+
+```rust
+TxScript WithdrawProtocolFee(invariant: Invariant, poolKey: PoolKey);
```

Comment:
I think it will be better to use same syntax for all entrypoints. 
`pub fn withdrawProtocolFee(poolKey: PoolKey) -> ();`